src/main/java/net/imagej/patcher/LegacyExtensions.java

```diff
@@ -763,13 +771,23 @@ private static void installOpenInterceptor(CodeHacker hacker) {
 						"else if (result instanceof java.lang.String) path = (java.lang.String)result;" +
 						"else return null; " +
 				"}");
-		hacker.replaceCallInMethod("ij.plugin.FolderOpener", "public void run(java.lang.String arg)", "ij.io.Opener", "openImage",
-			"Object result = ij.IJ._hooks.interceptOpenImage($1 + $2, -1);" +
-			"if (result != null && result instanceof ij.ImagePlus) {" +
-			"  $_ = (ij.ImagePlus)result;" +
-			"} else {" +
-			"  $_ = $proceed($$);" +
-			"}");
+		if (LegacyInjector.isImageJ1VersionAtLeast(hacker, "1.53m")) {
+			hacker.replaceCallInMethod("ij.plugin.FolderOpener", "public void run(java.lang.String arg)", "ij.io.Opener", "openTempImage",
```

Comment:
This change is due to 

https://github.com/imagej/ImageJ/commit/1b2324520b96ba0ea10ff76230d652c773cc8d9e#diff-143bc9f3e77f18608ffcfb8f16b3055fc811257c97fa06cdd1fe1a753170d10fL247-R247

We may want to contemplate intercepting `openImage` within `openTempImage` in the future:
https://github.com/imagej/ImageJ/blob/1b2324520b96ba0ea10ff76230d652c773cc8d9e/ij/io/Opener.java#L378